systemtest/src/main/resources/StrimziUpgradeST.json

```diff
@@ -63,7 +63,7 @@
       "logMessageVersion": "2.2"
     },
     "client": {
-      "beforeKafkaUpdate": "strimzi/test-client:0.11.4-kafka-2.1.0",
+      "beforeKafkaUpdate": "strimzi/test-client:0.12.1-kafka-2.1.0",
```

Comment:
isn't it pretty old?

Reply:
Yes, from my POV we don't need these tests anymore (for functionality testing), cause we will not provide any fixes. We can use it for "environment test" to see that environment is fine, because old tests were fine and only new are failing -> problem with upgrade

Reply:
I'm not sure how exactly did you meant this - but we had issues whihc were caused by how something looked several versions ago in and environment which went through the upgrade chain. So the upgrade chain IMHO needs to be maintained.

Reply:
so in that case we should add test which will go trough all versions and do upgrade chain (from version X to version Y, several upgrades in row). Currently, each test case deploy version X of CO and perform upgrade to version Y. After this whole env is deleted and next test case will deploy version Y and upgrade to version Z.